src/translate-once.js

```diff
@@ -13,7 +13,7 @@
   createDirective = function (attribute) {
     var namedDirective = getNamedDirectiveFromAttribute(attribute);
     angular.module(MODULE_NAME).directive(namedDirective, ['$parse', '$translate',
-    TranslateOnceAttributeDirective.bind(undefined, attribute)]);
+    angular.bind(undefined, TranslateOnceAttributeDirective, attribute)]);
```

Comment:
No problem, thanks for sharing this library!

And yeah, my teammates and I have always gone the route of including a file of essential polyfills for testing with PhantomJS—`angular.bind()` seems so verbose and less readable when sitting next to a usage of `Function.prototype.bind()`. But in this case, why not!